system-tests/tests/load/cre/workflow-don-load-test-streams-kind.toml

```diff
@@ -0,0 +1,172 @@
+duration = "2h" # specific to chaos test, ignore this value if you are performing a chaos test
+
+# ----- BLOCKCHAIN CONFIGURATION -----
+# Sets up a local Anvil blockchain for testing with 5-second block time
+[[blockchains]]
+chain_id = "1337"
+type = "anvil"
+docker_cmd_params = ["-b", "5"]
+
+# ----- BINARY PATHS -----
+# Path to the mock capability binary used for test simulation
+[binaries_config]
+mock_capability_binary_path = "../../../../../capabilities/bin/amd64/mock"
+
+# ----- JOB DISTRIBUTOR -----
+# Docker image for the job distributor service
+[jd]
+image = "localhost:5000/job-distributor:sha-707362c-amd64"
+
+# ----- MOCK CAPABILITIES -----
+# Trigger capability that simulates streams of data
+[[mock_capabilities]]
+name="streams-trigger"
+version= "2.0.0"
+type="trigger"
+description="streams trigger mock"
+
+# Target capability that simulates writing data to Ethereum
+[[mock_capabilities]]
+name="write_ethereum"
+version= "1.0.0"
+type="target"
+description="write trigger mock"
+
+# ----- TEST CONFIGURATION -----
+# Parameters for the Workflow DON load test
+[workflow_load]
+streams = 1000              # Number of streams to simulate
+jobs = 2                    # Number of jobs per stream
+
+# ----- INFRASTRUCTURE SETUP -----
+# Configures Docker-based test infrastructure
+[infra]
+type = "docker"
+
+# CRIB deployment configuration for local KIND cluster
+[infra.crib]
+namespace = "crib-workflow-don-load-test"
+folder_location = "../../../../../crib/deployments/cre"
+provider = "kind"           # Local Kubernetes in Docker
+
+# Team information for resource tracking
+[infra.crib.team_input]
+team="cre"
+product="workflow-don"
+component="workflow-don"
+cost_center="capabilities-df-migration-performance-tests"
+
+# ----- WORKFLOW NODES CONFIGURATION -----
+# Setup for a 6-node Workflow DON cluster (includes bootstrap node)
+[[nodesets]]
+nodes = 6
+override_mode = "all"           # Apply same configuration to all nodes
+http_port_range_start = 10100   # Starting port for HTTP endpoints
+name = "workflow"
+
+# Database configuration for the workflow nodes
+[nodesets.db]
+image = "postgres:12.0"
+port = 13000
+volume_name = "workflow_volume"
+
+# Chainlink node configuration for workflow nodes
+[[nodesets.node_specs]]
+[nodesets.node_specs.node]
+image = "localhost:5001/chainlink:develop"  # Local development image
+user_config_overrides = """
+        [Feature]
+        LogPoller = true
+        FeedsManager = true
+        [OCR2]
+        Enabled = true
+        DatabaseTimeout = '1s'
+        [P2P.V2]
+        Enabled = true
+        ListenAddresses = ['0.0.0.0:5001']
+      """
```

Comment:
this can be deleted from all configs, because when we generate the config we add that bit anyway